k8s-deploy/development/migration_job.yaml

```diff
@@ -27,11 +22,6 @@ spec:
                 secretKeyRef:
                   key: DEVISE_SECRET
                   name: pq-secrets
-            - name: DEVISE_SENDER
-              valueFrom:
-                secretKeyRef:
-                  key: DEVISE_SENDER
-                  name: pq-secrets
```

Comment:
I think this is a valid ENV VAR, we might want to set it to something sensible if it's not - sets the "from" email address for auth emails sent by Devise, e.g. password reset emails?  
![image](https://user-images.githubusercontent.com/1161161/212193689-221f2a75-ee22-4f51-b269-a3124d8f7fc5.png)

Reply:
It all goes through gov uk notify and the email address sent from is pq.tracker@notifications.service.gov.uk. The reply-to probably needs to be configured on Notify as it's a GSI address at the moment. I'll add a ticket for that. The Devise sender wasn't used before it was set to 'x'.